### ISSUE TITLE
Example text is not distinguished from definition text

### ISSUE THREAD

**Issue by mitchellevan** [w3c/wcag#634]
Expected behavior:

In WCAG 2.0, definition examples are prefixed with text such as "Example:" or "Example 1:". This prefix text makes it explicit that these are illustrations of the terms, not limitations to the normative definitions.

Issue and Impact:

In WCAG 2.1, only visual CSS styling (`class="example"`) distinguishes definition examples from definition text. This causes some confusion for me as a sighted user, because I must explore the definitions to figure out that the colored bar (#E0CB52) and indentation mean that these blocks of text are meant to be examples. In some cases this is an accessibility problem, such as the following, where the example appears in a screen reader as merely another paragraph of the definition.

Code Sample:

From WCAG 2.1 showing the issue

<pre><code>
&lt;dt&gt;&lt;dfn data-dfn-type="dfn" id="dfn-text-alternative"&gt;text alternative&lt;/dfn&gt;&lt;/dt&gt;
&lt;dd&gt;
    &lt;p&gt;&lt;a href="#dfn-text" class="internalDFN" data-link-type="dfn"&gt;Text&lt;/a&gt; that is programmatically associated with &lt;a href="#dfn-non-text-content" class="internalDFN" data-link-type="dfn"&gt;non-text content&lt;/a&gt; or referred to from text that is programmatically associated with non-text content.
      Programmatically associated text is text whose location can be programmatically determined
      from the non-text content.
    &lt;/p&gt;   
    &lt;p class="example"&gt;An image of a chart is described in text in the paragraph after the chart. The short
      text alternative for the chart indicates that a description follows.
    &lt;/p&gt;
    &lt;div class="note" id="issue-container-generatedID-133"&gt;&lt;div role="heading" class="note-title marker" id="h-note-133" aria-level="3"&gt;&lt;span&gt;Note&lt;/span&gt;&lt;/div&gt;&lt;p class=""&gt;...&lt;/p&gt;&lt;/div&gt;
&lt;/dd&gt;
</code></pre>

Screenshot of the above sample:

![Screenshot of the above code sample rendered in a browser](https://user-images.githubusercontent.com/3124805/53148248-ed8ab800-355e-11e9-8831-d1ab928299d9.png)

**Comment by alastc** [comment]
@michael-n-cooper is this a build-process thing?

**Comment by michael-n-cooper** [comment]
This is a respec issue. The [respec documentation for the example class](https://github.com/w3c/respec/wiki/example) says a heading is inserted, but evidently it isn't.

**Comment by jake-abma** [comment]
same issue here : https://github.com/w3c/wcag/issues/570

A PR might be a fix, if taken on, didn't have time yet...

> If you think a manual heading should be provided for every example in WCAG, then you could file a pull request on WCAG 2.1.

**Comment by mitchellevan** [comment]
@michael-n-cooper wrote:
> This is a respec issue. The [respec documentation for the example class](https://github.com/w3c/respec/wiki/example) says a heading is inserted, but evidently it isn't.

The respec documentation for the example class also says the example class is only for `pre` and `aside` elements, so according to respec this is working as designed.

A [respec comment in July 2014](https://github.com/w3c/respec/issues/272#issuecomment-50407796) suggested that the example class should also work for `p` paragraph elements.

So, is it the view of WCAG developers that the fix for #634 should be made in a respec enhancement request, a WCAG bugfix, or a combination of the two?

**Comment by awkawk** [comment]
Looks like the documentation requires that examples are wrapped in pre or aside elements. I suspect that the heading gets added correctly if we use that.

The notes we have don't feel like asides to me, but maybe they are. Examples seem a better fit for an aside, but given the visual appearance of notes and examples being so similar perhaps they should be the same?

**Comment by michael-n-cooper** [comment]
Michael to change elements of class="example" to <aside> unless they're already <pre>. Note respec does not make the source code of the example in heading in the same way as it does for notes, we may want to file an enhancement request.

**Comment by mitchellevan** [comment]
@shawna-slh This design issue is still occurring in the Glossary section of WCAG 2.1 and 2.2. If it makes sense to look again at this issue as part of docs redesign, would you please add the "docs redesign" label? The project does not let me label an issue.

**Comment by shawna-slh** [comment]
@mitchellevan Currently this is outside the scope of the docs redesign work, though there is a related issue. This point is more important, and hopefully it bubbles to the top of a priority soon.

**Comment by iadawn** [comment]
Closing as fixed
